lib/plugins/aws/provider.js

```diff
@@ -619,6 +623,7 @@ class AwsProvider {
               },
               additionalProperties: false,
             },
+            awsKmsKeyArn: { $ref: '#/definitions/awsKmsArn' },
```

Comment:
Is there a reason to add this? I think it's invalid property on provider and we already have `kmsKeyArn` defined for provider as available